ion-schema/src/isl/mod.rs

```diff
@@ -93,18 +94,22 @@ pub struct IslSchema {
     types: Vec<IslType>,
     // Represents all the inline IslImportTypes in this schema file.
     inline_imported_types: Vec<IslImportType>,
+    // Represents open content as `Element`s
+    open_content: Vec<Element>,
```

Comment:
This strategy means that we can't tell the difference between, e.g., 
```
$foo
$bar
type::{ name: foo, codepoint_length: 1 }
```
and
```
type::{ name: foo, codepoint_length: 1 }
$foo
$bar
```

If we're going to go with this strategy, we should document the limitations (in the rust doc) and the reason for the decision.

Reply:
Added a note for that in the doc comments.